### ISSUE TITLE
Jaws: ARIA Combobox with ARIA Textbox does not work with Virtual Cursor

### ISSUE THREAD

**Issue by Heumader** [FreedomScientific/VFO-standards-support#665]
### Summary
According to the ARIA 1.1 specification the following comboboxes should be valid:

`<span aria-label="My combobox label" role="combobox" aria-expanded="false">
<span role="textbox" aria-multiline="false" aria-label="My Textbox Label" tabindex="0">My Combobox Value</span>
</span>
`
Readonly combobox: 

`<span aria-label="My combobox label" role="combobox" aria-expanded="false">
<span role="textbox" aria-readonly="true" aria-multiline="false" aria-label="My Textbox Label" >My Combobox Value</span>
</span>
`

### Expected result

Jaws should identify this as valid combobox, reading labels, role, value and state(collapsed & readonly)

### Actual result

With virtual PC cursor enabled: Jaws reads the label, but omits the value. Instead of reading the combobox role,  Jaws uses the role of the previous focused element. E.g when a link was focused before the combobox, Jaws will read the combobox as a link.
Readonly: Jaws just reads: “My Combobox label” but nothing else

### Example

Codepen: [https://codepen.io/heumader/pen/BarEBaG](https://codepen.io/heumader/pen/BarEBaG)
A link and a button were added before and after the writeable combobox to demonstrate the false assignment of roles

It seems to me that JAWS only works well with the combobox-role if an <input> is used.  Other constructs using role="textbox" or role="searchbox" do not work very well. 

#### JAWS version and build number

Version 2022.2206.9
 
#### Operating System and version

Windows 10 Enterprise 21H2

#### Browser and version:

Chrome: 104.0.5112.102

**Comment by JAWS-test** [comment]
@Heumader 

This old combobox pattern  is obsolete. Please use the ARIA 1.2 pattern:
* https://www.w3.org/TR/wai-aria-1.2/#combobox
* https://www.w3.org/WAI/ARIA/apg/patterns/combobox/

Since the combobox pattern has changed 2 times in the last years (from ARIA 1 to 1.1 and 1.2) I think that all three patterns should be supported by JAWS, since many pages still contain the old patterns.

**Comment by Heumader** [comment]
Why was this closed? 
Jaws does not support any of the patterns (ARIA 1.0, ARIA 1.1 and ARIA 1.2) when anything other then an input html is used as combobox.  If an element with role="textbox" is used instead, it does not work in many browser screenreader combinations. 
Or was this fixed now?
